tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,82 @@
 import pytest
 import SCF
+import pickle
+import numpy as np
+
+Suv = pickle.load(open("suv.pkl", "rb"))

```

Comment:
These should not be defined globally in the test file, they should be accessed within each test through the fixture.